python/kfserving/kfserving/storage.py

```diff
@@ -26,22 +26,17 @@
 _GCS_PREFIX = "gs://"
 _S3_PREFIX = "s3://"
 _BLOB_RE = "https://(.+?).blob.core.windows.net/(.+)"
-_LOCAL_PREFIX = "file://"
 
 
 class Storage(object): # pylint: disable=too-few-public-methods
     @staticmethod
     def download(uri: str, out_dir: str = None) -> str:
         logging.info("Copying contents of %s to local", uri)
 
-        is_local = False
-        if uri.startswith(_LOCAL_PREFIX) or os.path.exists(uri):
-            is_local = True
+        if os.path.exists(uri):
+            return Storage._download_local(uri)
```

Comment:
Is that better to move this to following `Storage._download_blob(uri, out_dir)` as elif? and keep `Storage._download_local(uri, out_dir)` to handle user case that specify the `out_dir` option?

Reply:
Here I wish to make it return immediately if it's local path, and skip the mktemp step since and the raise exception part. 
For local path (which can only come from pvc now) as long as the path exist it should be successful or it will raise message in Storage._download_local to say "Local path %s does not exist.", the prefix type will never be touched.